README.md

```diff
@@ -1,6 +1,6 @@
 # Hound
 
-[![Build Status](https://travis-ci.org/hound-search/hound.svg?branch=master)](https://travis-ci.org/hound-search/hound) 
+![.github/workflows/go.yaml](https://github.com/spreadshirt/hound/workflows/.github/workflows/go.yaml/badge.svg)
```

Comment:
I think this isn't quite the right URL for our badge. This points to the badge that lives in a forked copy of Hound.